api/base/settings/defaults.py

```diff
@@ -27,6 +27,7 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = osf_settings.DEBUG_MODE
+DEBUG_PROPAGATE_EXCEPTIONS = True
```

Comment:
If you install the [django_extensions](http://django-extensions.readthedocs.org/en/latest/runserver_plus.html) package and remove this and run `python manage.py runserver_plus` you'll get werkzeug and it's interactive debugging console like you do with flask. There's also several other fancy things.